packages/form-validations/src/lang.ts

```diff
@@ -1,8 +1,20 @@
+/* eslint no-template-curly-in-string: "off" */
 import { Languages } from './types';
 
 export const lang: Languages = {
   en: {
     required: 'This field is required',
     default: 'Please enter a correct value',
+    email: 'This field requires a valid e-mail address',
+    number: 'This field requires a number',
+    integer: 'This field requires an integer value',
+    url: 'This field requires a valid website URL',
+    tel: 'This field requires a valid telephone number',
+    maxlength: 'This fields length must be <= ${1}',
+    minlength: 'This fields length must be => ${1}',
```

Comment:
this creates an arrow in my browser 😁 
how about we use the correct symbols? ≤ ≥

Reply:
Fixed.